### DISCUSSION TITLE
Couldn't create QR code: no data to encode during qrcodelink endpoint

### DISCUSSION THREAD

**Discussion by Vo-iD** [bbernhard/signal-cli-rest-api#476] [Q&A]
Hello! 

I've been working with signal-client-rest-api for a few hours and everything went well, but then I faced with some issues, API didn't work for me. I was struggling for a half of the day with it, so I decided to remove everything related to the api and start from the beginning. 
So, I went through [Getting started](https://github.com/bbernhard/signal-cli-rest-api/tree/master?tab=readme-ov-file#getting-started), created a folder, started docker container with latest version of the package, then tried to get QR code for ling API with my account and I see this response. 

HTTP GET: http://localhost:8080/v1/qrcodelink?device_name=cli
Response: 400 Bad Request, body: 
`{"error":"Couldn't create QR code: no data to encode"}`

What I'm doing wrong? 
I'll appreciate your support.

**Comment by bbernhard** [comment]
There is currently an upstream issue https://github.com/bbernhard/signal-cli-rest-api/issues/475

**Comment by mrarfarf** [comment]
Running into same issue with new install on Raspberry Pi 5. 

{"error":"Couldn't create QR code: no data to encode"}

Any ideas to help debug or fix? Saw the upstream mention from Feb 5, but the steps to fix that way are over my head.. Thanks for any help :)
